crates/forge/src/lib.rs

```diff
@@ -76,6 +78,11 @@ enum ForgeSubcommand {
         /// Name of a new project
         name: String,
     },
+    /// Create a new Forge project at <PATH>
```

Comment:
`PATH` here is a bit reminiscent of `$PATH` environment variable.
```suggestion
    /// Create a new Forge project at <path>
```

Reply:
I see, but this is how it will be shown after `snforge new --help`:
```
Usage: scarb new [OPTIONS] <PATH>
```
Scarb has a similar description for the `scarb new` command:
```
Create a new Scarb package at <PATH>
```

Reply:
Ok I see, let's leave it then, it's not a big thing.

Reply:
Ok I see, let's leave it then, it's not a big thing.